tools/wptrunner/wptrunner/formatters/chromium.py

```diff
@@ -157,17 +162,19 @@ def test_status(self, data):
     def test_end(self, data):
         test_name = data["test"]
         actual_status = self._map_status_name(data["status"])
+        expected_statuses = self._get_expected_status_from_data(actual_status, data)
```

Comment:
I want to confirm my understanding by highlighting one scenario, and you can tell me if thats expected or not. In the case of `actual_status` is `PASS` and `data["expected"]` is `None` on line 165, which means the absence of "expected" implies a PASS is expected by WPT Runner. In other words, `expected_statuses` is `[PASS]` and no regression from WPT Runner. However, when there is a subtest failure, `actual_status` is FAIL on line 170, which means `cur_dict["is_regression"] = True` on line 89. In other words, a regression on Chromium CQ. Is this inconsistency between CQs and WPT Runner expected?

Reply:
Yes this is expected.
For this case, Chromium CQ only reports one status while WPTRunner reports two (PASS for test, FAIL for subtest). So we combine WPT's "test pass" + "subtest fail" into a single "FAIL" status for Chromium.

Logically this makes sense - the test ran and something didn't work, so it's both unexpected and a regression.

Reply:
Thanks for the explanations!